### PR TITLE
Search engine choice

### PR DESCRIPTION
**Pull request by chrmod**
First screen from Search Engine Choice experience

It implements:
* shows once as onboarding
* allow user to select default search engine
* search engine set with experiment API

Screenshot 2020-10-15 at 17 11 42

### CODE REVIEW

File: src/assets/GhoserySearch.svg [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sammacbeth) [inline src/assets/GhoserySearch.svg]: Typo in the file name

File: src/experiment_apis/ghostery_schema.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[{
+  "namespace": "ghostery",
+  "description": "API giving overring various low level functionalities",
```
Comment (sammacbeth) [inline src/experiment_apis/ghostery_schema.json]:
```suggestion
  "description": "API to change the default search engine",
```

File: src/pages/choice-screen.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  <head>
+    <title>Search Engine Choice</title>
+    <meta charset="UTF-8" />
+    <script src="common.js"></script>
+    <style>
+      @font-face {
+        font-family: Roboto;
+        src: url('../fonts/Roboto-Regular.ttf');
+        font-weight: 400;
+      }
```
Comment (sammacbeth) [inline src/pages/choice-screen.html:12]: Can you split CSS into a separate file?

File: src/pages/choice-screen.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    </style>
+  </head>
+  <body>
+    <section class="header">
+      <img id="logo" src="../assets/GhosteryBrowser.svg" />
+    </section>
+    <section class="content">
+      <h1>Welcome! Choose you default search</h1>
+      <h2>Pick a default search engine for all your searches.</h2>
```
Comment (sammacbeth) [inline src/pages/choice-screen.html:130]: Can we use `i18n` already for these strings so it will be easier to localize later?

**Review by sammacbeth — COMMENTED** [review]
Looks good. Just a couple of nits

File: src/pages/choice-screen.html (hunk 3 above)
Comment (chrmod) [reply src/pages/choice-screen.html:12]: Will do that when implementing next page

File: src/pages/choice-screen.html (hunk 4 above)
Comment (chrmod) [reply src/pages/choice-screen.html:130]: make sense. Will do it in a follow up

**Pull request merged**